metrics/web/test/base/presentation/widgets/base_popup_test.dart

```diff
@@ -136,40 +136,43 @@ void main() {
     );
 
     testWidgets(
-      "closes a popup, when tap on the popup's empty space if the closePopupOnEmptySpaceTap is true",
+      "closes a popup when tap outside of the popup if the closeOnTapOutside is true",
       (WidgetTester tester) async {
         await tester.pumpWidget(
           _BasePopupTestbed(
             popup: testPopupWidget,
             triggerBuilder: _defaultTriggerBuilder,
-            closePopupOnEmptySpaceTap: true,
+            closeOnTapOutside: true,
+            offsetBuilder: (Size childSize) {
+              return Offset(childSize.width, childSize.height);
+            },
           ),
         );
 
         await tester.tap(triggerWidgetFinder);
         await tester.pumpAndSettle();
-        await tester.tap(popupWidgetFinder);
+        await tester.tap(triggerWidgetFinder);
         await tester.pumpAndSettle();
 
         expect(popupWidgetFinder, findsNothing);
       },
     );
 
     testWidgets(
-      "does not close a popup, when tap on the popup's empty space if the closePopupOnEmptySpaceTap is false",
+      "does not close a popup when tap outside of the popup if the closeOnTapOutside is false",
       (WidgetTester tester) async {
         await tester.pumpWidget(
           _BasePopupTestbed(
             popup: testPopupWidget,
             triggerBuilder: _defaultTriggerBuilder,
-            closePopupOnEmptySpaceTap: false,
+            closeOnTapOutside: false,
           ),
         );
 
         await tester.tap(triggerWidgetFinder);
         await tester.pumpAndSettle();
-        await tester.tap(popupWidgetFinder);
-        await tester.pumpAndSettle();
+        await tester.tapAt(Offset.infinite);
```

Comment:
Would it be more reliable? 
```suggestion
        await tester.tap(triggerWidgetFinder);
```
or
```suggestion
        await tester.tapAt(find.byType(Scaffold));
```